src/main/kotlin/pl/edu/uj/ii/ksi/mordor/configuration/TesseractConfiguration.kt

```diff
@@ -0,0 +1,16 @@
+package pl.edu.uj.ii.ksi.mordor.configuration
+
+import net.sourceforge.tess4j.Tesseract
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class TesseractConfiguration {
+    @Bean
+    fun tesseract(): Tesseract {
+        val tesseract = Tesseract()
+        tesseract.setDatapath("./src/main/resources/")
```

Comment:
check if it works when mordor is bundled in a release jar file

Reply:
I've checked, it works